static/index.html

```diff
@@ -247,7 +253,7 @@
                     </div>
                 </li>
                 <li>
-                    <label>Supports (mod has specific code to relating to these mods)<br /><textarea id="supports" cols="40"></textarea></label>
+                    <label>Supports (mod has specific code to relating to these mods or is known to play well with them)<br /><textarea id="supports" cols="40"></textarea></label>
```

Comment:
> or is known to play well with them
Any 2 mods that don't have an explicit `conflicts` set have this type of implicit `supports`. See https://github.com/KSP-CKAN/CKAN/issues/1574#issuecomment-183704091 for a more complete explanation.